From d2b3a55bf679ed7a54e747b8daed76ffd6c2579b Mon Sep 17 00:00:00 2001
From: Tirumalesh Chalamarla <tchalamarla@caviumnetworks.com>
Date: Tue, 16 Oct 2018 12:49:11 +0300
Subject: [PATCH 0264/1051] octeontx-mbox: fix the gap in mbox return code

fix the gap in MBOX definitions for return values

Signed-off-by: Tirumalesh Chalamarla <tchalamarla@caviumnetworks.com>
Signed-off-by: Jerin Jacob <jerin.jacob@caviumnetworks.com>
Signed-off-by: Santosh Shukla <santosh.shukla@caviumnetworks.com>
Signed-off-by: Yury Norov <ynorov@marvell.com>
Signed-off-by: Yury Norov <ynorov@caviumnetworks.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/cavium/octeontx-83xx/octeontx_mbox.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/cavium/octeontx-83xx/octeontx_mbox.h b/drivers/net/ethernet/cavium/octeontx-83xx/octeontx_mbox.h
index 138946b6eed1..ae29fca907ef 100644
--- a/drivers/net/ethernet/cavium/octeontx-83xx/octeontx_mbox.h
+++ b/drivers/net/ethernet/cavium/octeontx-83xx/octeontx_mbox.h
@@ -45,8 +45,8 @@ enum coproc_t {
 
 /*resp messages*/
 #define MBOX_RET_SUCCESS	0x0
-#define MBOX_RET_INVALID	0x2
-#define MBOX_RET_INTERNAL_ERR	0x3
+#define MBOX_RET_INVALID	0x1
+#define MBOX_RET_INTERNAL_ERR	0x2
 
 /* magic values used for normal and oob data notification */
 #define MBOX_TRIGGER_NORMAL	0x00 /* normal msg transport */
-- 
2.17.1

